<ui version="4.0" >
 <class>WASAPIConfig</class>
 <widget class="QWidget" name="WASAPIConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Device selection</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Input (Microphone)</string>
        </property>
        <property name="buddy" >
         <cstring>qcbInputDevice</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="qcbInputDevice" >
        <property name="toolTip" >
         <string>Device to use for microphone</string>
        </property>
        <property name="whatsThis" >
         <string>This set which source mumble should use for the microphone. The &lt;i>default&lt;/i> device means to use the PulseAudio defaults.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Output</string>
        </property>
        <property name="buddy" >
         <cstring>qcbOutputDevice</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="qcbOutputDevice" >
        <property name="toolTip" >
         <string>Device to use for speakers/headphones</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QCheckBox" name="qcbEcho" >
        <property name="toolTip" >
         <string>Enable echo cancellation</string>
        </property>
        <property name="whatsThis" >
         <string>This enables echo cancellation. Don't enable this unless you are listening on speakers; for headphones it will cause audio degradation.</string>
        </property>
        <property name="text" >
         <string>Use echo cancellation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
